sdk/core/core-http/src/policies/proxyPolicy.ts

```diff
@@ -14,8 +14,7 @@ import { Constants } from "../util/constants";
 import { URLBuilder } from "../url";
 import { getEnvironmentValue } from "../util/utils";
 
-let noProxyList: string[] = [];
-let isNoProxyInitalized = false;
+export const noProxyList: string[] = loadNoProxy();
```

Comment:
Nit: We might need to decorate the newly exported symbols with the `@internal` doc tag so they don't show up in docs.

Reply:
Added